assets/queries/openAPI/responses_object_is_empty/query.rego

```diff
@@ -0,0 +1,35 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+options := {null, {}}
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+
+	doc.paths[path][op].responses == options[x]
+
+	result := {
+		"documentId": doc.id,
+		"searchKey": sprintf("paths.{{%s}}.{{%s}}.responses", [path, op]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("paths.{{%s}}.{{%s}}.responses is not empty", [path, op]),
+		"keyActualValue": sprintf("paths.{{%s}}.{{%s}}.responses is empty", [path, op]),
+	}
+}
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+
+	doc.components.responses == options[x]
```

Comment:
Since responses in components a map, shouldn't we add another verification to check if every entry in responses object map are not empty as well ex:
```
components:
  responses:
    Success:
    Error:
      ... 
```
Where in this case the Success entry is empty but the Error is not, I believe the Success entry should give a result as well

Reply:
Responses Object is a map of response object in components. Since this query is related to check if 'responses' object is empty, is it relevant to check for the 'response' object as well?